sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -9094,6 +9406,139 @@ prefetch(AccessorT acc, PropertyListT props = {}) {
   prefetch<T, VS>(acc, 0, Mask, props);
 }
 
+/// template <typename T, int BlockWidth, int BlockHeight = 1, int NBlocks = 1,
+///          bool Transposed = false, bool Transformed = false,
+///          typename PropertyListT = empty_properties_t,
+///          int N = detail::get_lsc_block_2d_data_size<
+///              T, NBlocks, BlockHeight, BlockWidth, Transposed,
+///              Transformed>()>
+/// simd<T, N>
+/// load_2d(const T *Ptr, unsigned SurfaceWidth, unsigned SurfaceHeight,
+///             unsigned SurfacePitch, int X, int Y,
+///             PropertyListT props = {});
+/// 2D USM pointer block load.
+/// Supported platforms: PVC
+/// VISA instruction: lsc_load_block2d.ugm
+///
+/// Collects elements located at specified address and returns them
+/// as a single \ref simd object.
+///
+/// @tparam T is element type.
+/// @tparam BlockWidth is the block width in number of elements.
+/// @tparam BlockHeight is the block height in number of elements.
+/// @tparam NBlocks is the number of blocks.
+/// @tparam Transposed is the transposed version or not.
+/// @tparam Transformed is apply VNNI transform or not.
+/// @tparam N is the data size
```

Comment:
Here we skip `PropertyListT` which I expect but it comes in between Transformed and N, so if a user tries to follow this doc for a fcn call it will fail, maybe we should move PropertyListT to the last template param in the actual call?

Reply:
It also will make load_2d/prefetch_2d consistent with store_2d, which already has `PropertyListT` placed after `N`